packages/openbridge-webcomponents/src/navigation-instruments/readout-input/readout-input.css

```
@@ -0,0 +1,261 @@
+    display: grid;
+    justify-items: end;
+    align-items: center;
+
+    .degree-template {
+      grid-area: 1 / 1;
+    }
+
+    .degree,
+    .degree-template {
+      @mixin font-instrument-value-m-active;
+      display: block;
```

Stylelint (17.4.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)